linux/library/src/flutter_embedder.cc

```diff
@@ -81,13 +81,15 @@ static void GLFWOnFlutterPlatformMessage(const FlutterPlatformMessage *message,
     return;
   }
   GLFWwindow *window = reinterpret_cast<GLFWwindow *>(user_data);
-  Json::Reader parser;
+  Json::CharReaderBuilder rbuilder;
+  std::unique_ptr<Json::CharReader> const parser(rbuilder.newCharReader());
   Json::Value json;
+  std::string parse_errors;
   std::string raw_string(reinterpret_cast<const char *>(message->message), 0,
                          message->message_size);
-  bool parsing_successful = parser.parse(raw_string, json);
+  bool parsing_successful = parser->parse(raw_string.c_str(), raw_string.c_str() + raw_string.size(), &json, &parse_errors);
```

Comment:
This and 92 will need to be wrapped to follow the C++ style guide length limits.

https://google.github.io/styleguide/cppguide.html

But it also doesn't seem like there's anything to be gained by round-tripping through std::string with this API.